plugins/Jobs.cpp

```diff
@@ -661,6 +661,13 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
         const list<string> nameList = SParseList(request["name"]);
         string safeNumResults = SQ(max(request.calc("numResults"),1));
         bool mockRequest = command.request.isSet("mockRequest") || command.request.isSet("getMockedJobs");
+        bool orderByCreated = command.request.isSet("orderByCreated") ? request.test("orderByCreated") : false;
```

Comment:
request.test will handle checking if it's set and setting it to false I believe, so the ternary is redundant